services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/structure/file/FileOnDisk.java

```diff
@@ -60,6 +60,6 @@ public void setBytes(byte[] bytes) {
    * Does nothing.
    */
   @Override
-  public void prepare(ImmutableStack<Object> indices) {
+  public void prepare(ImmutableStack<Object> indices) throws UnsupportedOperationException{
```

Comment:
The issue there is the *empty method*, it is still empty.

> Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation.

Use `throw new UnsupportedOperationException()` (not sure if this is intended here) or add a comment why this is empty.